policyengine_us/parameters/gov/states/il/tax/income/exemption/cap.yaml

```
@@ -0,0 +1,21 @@
+description: IL Exemption Cap
```

Could you also rename the file to `income_limit.yaml`?
```suggestion
description: Maximum AGI to be eligible for Illinois exemptions.
```